public/js/index.js

```diff
@@ -1264,20 +1264,29 @@ function initWikiForm() {
             forceSync: true,
             previewRender: function (plainText, preview) { // Async method
                 setTimeout(function () {
-                    // FIXME: still send render request when return back to edit mode
-                    $.post($editArea.data('url'), {
-                            "_csrf": csrf,
-                            "mode": "gfm",
-                            "context": $editArea.data('context'),
-                            "text": plainText
-                        },
-                        function (data) {
-                            preview.innerHTML = '<div class="markdown ui segment">' + data + '</div>';
-                            emojify.run($('.editor-preview')[0]);
-                        }
-                    );
-                }, 0);
-
+                    let $toolbar = $(preview).closest('.CodeMirror-wrap').prev();
```

Comment:
```suggestion
                    const $toolbar = $(preview).closest('.CodeMirror-wrap').prev();
```
Suggestion by as per linter error.